### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by SekoViper**
# Summary
## Implement the following: 

- Create Rails API app 
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table

Frontend [link](https://github.com/SekoViper/hello-react-front-end)
Frontend PR [link](https://github.com/SekoViper/hello-react-front-end/pull/3)

### CODE REVIEW

File: app/controllers/greetings_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# class GreetingsController < ApplicationController
-#   def index
-#     @greeting = Message.order('RANDOM()').first.greeting
-#     render json: { greeting: @greeting }
-#   end
-# end
+class GreetingsController < ApplicationController
```
Comment (gemmen29) [inline app/controllers/greetings_controller.rb:1]: - [ ] Kindly fix this error when I run both projects I get this error
image

# Hint:
- You can fix it by enabling cors in the backend repo.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @SekoViper, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/greetings_controller.rb (hunk 1 above)
Comment (Whoistolu) [reply app/controllers/greetings_controller.rb:1]: - [ ] Well resolved 👍🏽 
Screenshot 2023-05-06 at 18 14 01

**Review by Whoistolu — APPROVED** [review]
### Hi @SekoViper 

Great effort implementing the recommended changes by the previous developer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.
- [ ] [OPTIONAL] Kindly remove the broken link of the Live Demo section of this project since this project has not been deployed yet. You can attach it once the project gets deployed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**